JS-108_javascript-basics/exercise/guess-the-number/js/app.js

```diff
@@ -0,0 +1,29 @@
+// Guess the number
+let guesses = 0;
+let answer;
+let number;
+let gameOver = false;
+
+alert('I am thinking of a number from 1 to 20.');
+
+number = Math.ceil(Math.random() * 20);
+console.log(`number: ${number}`);
+
+while ( ! gameOver && guesses < 3 ) {
+  answer = prompt("Guess the number");
+  guesses++;
+
+  console.log(`${number}, ${answer}`);
+
+  if ( +answer === number ) {
```

Comment:
Although this works, this is not an acceptable industry standard. It would be better to use `Number()`, `Boolean()`, `String()`, etc. global functions when casting a variable to a particular data type.
```suggestion
  if ( Number(answer) === number ) {
```

Reply:
This is noted.